website/resources/views/errors/status.blade.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+status(
+"Success" => "Status: Success"
```

Comment:
There is no need for this file, we can just return json of `[ "success" => true ]` or false if it failed.